Makefile.am

```diff
@@ -82,7 +82,9 @@ ipfixprobe_src=main.cpp \
 		ipfix-basiclist.cpp \
 		ipfix-basiclist.h \
 		basicplusplugin.h \
-		basicplusplugin.cpp
+		basicplusplugin.cpp \
+		wgplugin.h \
+		wgplugin.cpp
 
 
```

Comment:
Missing `traffic-samples/wg-sample.pcap` in `EXTRA_DIST` variable.

Reply:
Ah, good catch, will fix.